src/screens/toDoList/ToDoList.tsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
-import {FlatList, ListRenderItem, Modal, Text, View} from 'react-native';
+import {Modal, Text, View} from 'react-native';
 import ToDoListRow from './components/ToDoListRow';
```

Comment:
```suggestion
import {ToDoListRow} from './components';
```
export ToDoListRow from index.ts